migrations/20240130125545-add-expense-payment-method.js

```diff
@@ -0,0 +1,96 @@
+        AND pm."service" = 'wise' AND pm.type = 'bank_transfer' AND pm."CollectiveId" = e."HostCollectiveId"
+      RETURNING e.*;
+    `);
+    console.info(metadata.rowCount, 'expenses updated with wise payment method');
+
+    [, metadata] = await queryInterface.sequelize.query(`
+      WITH stripevirtualcard AS (SELECT DISTINCT e."HostCollectiveId" FROM "Expenses" e WHERE e."createdAt" > '2024-01-01' AND e.status = 'PAID' AND e.data#>>'{transaction,card}' IS NOT NULL)
+      INSERT INTO "PaymentMethods" ("service", "type", "CollectiveId", "saved", "data")
+      SELECT 'stripe' as "service", 'virtual_card' as "type", "HostCollectiveId" as "CollectiveId", False as "saved", '{ "migration": "20240130125545-add-expense-payment-method" }'::JSONB as "data" FROM stripevirtualcard ON CONFLICT DO NOTHING;
+    `);
+    console.info(metadata.rowCount, 'stripe virtual card payment methods created');
+    [, metadata] = await queryInterface.sequelize.query(`
+      UPDATE "Expenses" as e
+      SET "PaymentMethodId" = pm.id, "data" = '{ "migration": "20240130125545-add-expense-payment-method" }'::JSONB || e.data
+      FROM "PaymentMethods" pm
+      WHERE
+        e."deletedAt" IS NULL AND e."createdAt" > '2024-01-01' AND e.status = 'PAID' AND e.data#>>'{transaction,card}' IS NOT NULL
+        AND pm."service" = 'stripe' AND pm.type = 'virtual_card' AND pm."CollectiveId" = e."HostCollectiveId"
+      RETURNING e.*;
+    `);
+    console.info(metadata.rowCount, 'expenses updated with stripe virtual card payment method');
+
+    [, metadata] = await queryInterface.sequelize.query(`
+      WITH paypalpayouts AS (SELECT DISTINCT e."HostCollectiveId" FROM "Expenses" e WHERE e."createdAt" > '2024-01-01' AND e.status = 'PAID' AND e.data#>>'{payout_batch_id}' IS NOT NULL)
+      INSERT INTO "PaymentMethods" ("service", "type", "CollectiveId", "saved", "data")
+      SELECT 'paypal' as "service", 'payout' as "type", "HostCollectiveId" as "CollectiveId", False as "saved", '{ "migration": "20240130125545-add-expense-payment-method" }'::JSONB as "data" FROM paypalpayouts ON CONFLICT DO NOTHING;
+    `);
+    console.info(metadata.rowCount, 'paypal payout payment methods created');
+    [, metadata] = await queryInterface.sequelize.query(`
+      UPDATE "Expenses" as e
+      SET "PaymentMethodId" = pm.id, "data" = '{ "migration": "20240130125545-add-expense-payment-method" }'::JSONB || e.data
+      FROM "PaymentMethods" pm
+      WHERE
+        e."deletedAt" IS NULL AND e."createdAt" > '2024-01-01' AND e.status = 'PAID' AND e.data#>>'{payout_batch_id}' IS NOT NULL
+        AND pm."service" = 'paypal' AND pm.type = 'payout' AND pm."CollectiveId" = e."HostCollectiveId"
+      RETURNING e.*;
+    `);
+    console.info(metadata.rowCount, 'expenses updated with paypal payout payment method');
+
+    [, metadata] = await queryInterface.sequelize.query(`
```

Comment:
Is this necessary?

Reply:
As we should have adaptive paymentMethods

Reply:
There are 4 adaptive payment methods created.